modules/starfish/2.0/config/default.yaml

```diff
@@ -10,12 +10,22 @@ lcr-modules:
         inputs:
             
             vcf: 
-                # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
-                # Each key:value pair is the name of the variant caller and the path to its VCF file
-                # Each key should be unique in this set. 
-                # Note: If you aren't using strelka as one of the inputs to this, you will have to either delete the
-                # following line or copy the entire config to your project and omit this line from your project config. 
-                strelka: "__UPDATE__"
+
```

Comment:
I'm sure I had commented here too. Is this change to help ensure consistency in what order the tools get combined by starfish?

Reply:
It is indeed. With Strelka as the first key here, it was always loaded first even if a second config was loaded afterwards that clobbered the UPDATE string.